ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/home/view/HomeView.kt

```diff
@@ -179,7 +179,7 @@ private fun DebugMenu(resetSdk: () -> Unit, wipeEntireWallet: () -> Unit) {
 private fun HomeMainContent(
     paddingValues: PaddingValues,
     walletSnapshot: WalletSnapshot,
-    transactionHistory: List<Transaction>,
+    transactionHistory: List<CommonTransaction>,
```

Comment:
It works for now, but this might cause trouble later.  Currently, Synchronizer has APIs like getMemo() and getRecipient() that require the TransactionOverview.  You won't be able to do that with these CommonTransaction objects.

How do you think we might be able to make that work better?

Reply:
Understand. So in that case another simple solution for our current use case - just printing transactions information and of course later drawing them to a better UI (as designs propose) could be having the `CommonTransaction` as a sealed class. It can then hold both types. See the last commit. What do you think? Thank you.

Reply:
Yes that solution is reasonable.  Hopefully we can move that kind of API into the SDK in the coming weeks.